addon/content/conversation.css

```diff
@@ -772,28 +768,37 @@ html[dir="rtl"] .attachmentsDropDown .arrow {
   background-color: #68ab57 !important;
 }
 
-/* DKIM Verifier plugin */
-.special-tag.dkim-signed.SUCCESS {
+/* General pills */
+.special-tag.success {
   background-color: #68ab57 !important;
 }
 
-.special-tag.dkim-signed.SUCCESS.warnings {
+.special-tag.warning {
   background-color: orange !important;
 }
 
-.special-tag.dkim-signed.PERMFAIL {
+.special-tag.error {
   background-color: red !important;
 }
 
-.dkim-signed {
+.special-tag-ext-icon {
+  margin-inline-end: 0.2em;
+}
+
+.messageHeader img {
+  /* Override ".messageHeader img" to default ".icon" */
+  vertical-align: -0.4em;
+}
```

Comment:
A little dirty workaround, as I do not know to which other elements `.messageHeader img` is applied, and if the `vertical-align: text-top` is wanted there.

Reply:
This doesn't actually appear to be used, since the pill is in `.messageBody`. I'm going to merge for now, remove both of the `.messageHeader img` options (there's no `img`s in `.messageHeader`), and we can always tweak in a follow-up.